lib/chef/provider/package/snap.rb

```diff
@@ -137,7 +137,7 @@ def call_snap_api(method, uri, post_data = nil?)
 
           # while it is expected to allow clients to connect using https over
           # a tcp socket, at this point only a unix socket is supported. the
-          # socket is /run/snapd.socket note - unixsocket is not defined on
+          # socket is /run/snapd.socket note - unix socket is not defined on
```

Comment:
Very nitpicky, but I think referring to the class name is appropriate here.
```suggestion
          # socket is /run/snapd.socket note - UNIXSocket is not defined on
```